src/OpenTelemetry.Instrumentation.GrpcNetClient/TracerProviderBuilderExtensions.cs

```diff
@@ -28,16 +28,21 @@ public static class TracerProviderBuilderExtensions
         /// Enables gRPClient Instrumentation.
         /// </summary>
         /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+        /// <param name="options">GrpcClient configuration options.</param>
         /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
         public static TracerProviderBuilder AddGrpcClientInstrumentation(
-            this TracerProviderBuilder builder)
+            this TracerProviderBuilder builder,
+            Action<GrpcClientInstrumentationOptions> options = null)
```

Comment:
Rename `options` to `configure`.

ref: https://github.com/dotnet/runtime/blob/f1f7681464bc8e6662e15242d3babca75c914565/src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerExtensions.cs#L82